components/var_dumper/introduction.rst

```diff
@@ -62,6 +62,17 @@ current PHP SAPI:
     You'll also learn how to change the format or redirect the output to
     wherever you want.
 
+.. tip::
+
+    In order to have the ``dump()`` function always available, you can install
+    it globally in your dev environment:
+
+    #. Run ``composer global require symfony/var-dumper``;
+    #. Add ``auto_prepend_file = ${HOME}/.composer/vendor/autoload.php``
+       to your ``php.ini`` file;
```

Comment:
Does PHP interpret environment variables in the `php.ini` file? What happens if the file doesn't exist?